__tests__/app.test.js

```diff
@@ -48,3 +48,38 @@ describe("GET /api/", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id", () => {
+  test("200: should return the article object linked to the given id", () => {
+    return request(app)
+      .get("/api/articles/3")
+      .expect(200)
+      .then(({ body }) => {
+        expect(typeof body.article).toBe("object");
```

Comment:
You ideally want to write your test so it checks a particular article id, for example - article_id: 3 👍 How could you refactor this so you are using .toMatchObject?